jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/test/java/org/jbpm/workbench/ht/client/editors/taskslist/AbstractTaskListPresenterTest.java

```diff
@@ -225,7 +226,7 @@ public void releaseTaskTest() {
 
     @Test
     public void claimTaskTest() {
-        final TaskSummary task = TaskSummary.builder().id(TASK_ID).deploymentId(TASK_DEPLOYMENT_ID).build();
+        final TaskSummary task = TaskSummary.builder().id(TASK_ID).deploymentId(TASK_DEPLOYMENT_ID).status("Ready").build();
 
         getPresenter().claimTask(task);
 
```

Comment:
you can probably check what is executed in the callback as well.

Reply:
I am trying。